Web/stockdogweb/src/containers/Stock/Stock.js

```diff
@@ -5,16 +5,26 @@ import Navbar from '../../components/Navigation/Navbar';
 import Graph from '../../components/Graph/Graph';
 import Article from '../../components/Article/Article';
 import Trade from '../../components/Trade/Trade';
+import Transaction from '../../components/Trade/Transaction/Transaction';
 
 class Stock extends Component {
    constructor(props) {
       super(props);
       this.state = {
          title: "About",
-         content: "Advanced Micro Devices, Inc. engages in the provision of semiconductor businesses. It operates through the Computing and Graphics and Enterprise, Embedded and Semi-Custom segments."
+         content: "Advanced Micro Devices, Inc. engages in the provision of semiconductor businesses. It operates through the Computing and Graphics and Enterprise, Embedded and Semi-Custom segments.",
+         transactionIsOpen: false
       };
    }
 
+   activeTransactionModal = () => {
```

Comment:
Not necessary but might want to name these something with a verb like "activateTransactionModal" and "deactivateTransactionModal" since it's changing the state.

Reply:
Yikes I can't read